src/frontend/components/data-table/lib/component.js

```diff
@@ -323,6 +349,11 @@ class DataTableComponent extends Component {
     })
   }
 
+  getApiEndpoint(title) {
+    const table = window.location.pathname.substring(1).split('/')[0];
+    return `/api/${table}/fields?title=${title}&search=`;
```

Comment:
There should be a better way of getting the table name. One option might be to use `$(body).data("layout-identifier")`

Reply:
I didn't even spot that in the code, yet another useful titbit to add to my log! :smile: